src/dswx_sar/region_growing.py

```diff
@@ -0,0 +1,329 @@
+import os
+import logging
+import time
+import mimetypes
+
+import numpy as np
+from scipy import ndimage
+from joblib import Parallel, delayed
+
+from dswx_sar import dswx_sar_util
+from dswx_sar.dswx_runconfig import RunConfig, _get_parser
+
+
+logger = logging.getLogger('dswx_s1')
+
+
+def region_growing(likelihood_image,
+                   initial_seed=0.6,
+                   relaxed_threshold=0.45,
+                   maxiter=200):
+    """The regions are then grown from the seed points to adjacent
+    points since it covers the relaxed_threshold values.
+
+    Parameters
+    ----------
+    likelihood_image : numpy.ndarray
+        fuzzy image with values [0, 1]representing
+        likelihood of water where 0 is 0% and 1 is 100%
+    initial_seed : float
```

Comment:
What do you think of either `seed_threshold` or `initial_threshold` to replace `initial_seed`? Without needing to look at the docstring, I think either suggestion is more specific/descriptive and there's a naming consistency that implies a relation to `relaxed_threshold`.